packages/components/root/src/root.css

```diff
@@ -31,6 +31,7 @@
 	-ms-grid-columns: 0 minmax(var(--ts-aside), 1fr) 0;
 	grid-template-columns: 0 minmax(var(--ts-aside), 1fr) 0;
 }
+
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this usage of the experimental "host" pseudo. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.ts.sv/coding_rules#rule_key=css%3Aexperimental-pseudo-usage)